backend/FwLite/FwLiteShared/FwLiteSharedKernel.cs

```diff
@@ -64,6 +71,26 @@ private static void AddAuthHelpers(this IServiceCollection services, IHostEnviro
         }
     }
 
+    private static void ConfigureHttpClientChaos(IHttpClientBuilder builder)
+    {
+        builder.AddResilienceHandler("chaos",
```

Comment:
This seems weird to me. We're only adding resiliency if chaos is enabled. So, the chaos isn't testing our "real" app. Should we just always add resilience?